sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/AnalyzeSentimentActionResultPropertiesHelper.java

```diff
@@ -31,8 +32,8 @@ public static void setAccessor(final AnalyzeSentimentActionResultAccessor analyz
         accessor = analyzeSentimentActionResultAccessor;
     }
 
-    public static void setDocumentResults(AnalyzeSentimentActionResult actionResult,
-        AnalyzeSentimentResultCollection documentResults) {
-        accessor.setResult(actionResult, documentResults);
+    public static void setDocumentsResults(AnalyzeSentimentActionResult actionResult,
```

Comment:
Is this change consistent across all languages?
Also, since this is a public API, its a potential breaking change?

Reply:
This changes is consistent cross all languages. we had a renaming meeting to address it.